README.md

```diff
@@ -18,9 +18,18 @@ If you use RStudio, you can press Cmd/Ctrl + Shift + B to run make.
 
 ## Installing dependencies
 
-There are a number of dependencies required to build this book. We have a full installation recipe for ubuntu 12.04 [here](https://github.com/hadley/ggplot2-book/blob/master/.travis.yml). Even if you run a different operating system, please look there for R package requirements. Most of the R packages are available on CRAN via `install.packages` except for [bookdown](https://github.com/hadley/bookdown) and [captioner](https://github.com/adletaw/captioner). You can install both with:
+There are a number of dependencies required to build this book. We have a full installation recipe for Ubuntu 12.04 [here](https://github.com/hadley/ggplot2-book/blob/master/.travis.yml). Even if you run a different operating system, please look there for R package requirements. 
+All of the R packages are available on CRAN via `install.packages` except for [bookdown](https://github.com/hadley/bookdown) and [captioner](https://github.com/adletaw/captioner). You can install both
+sets of packages with:
 
 ```r
+pkgs <- c("lubridate", "rvest", "magrittr", "gridExtra", "dplyr", "plyr",
+          "tidyr", "xtable", "nlme", "effects", "broom", "hexbin", "maps",
+          "maps", "Hmisc", "devtools")
+reqs <- as.numeric(lapply(pkgs, require, character.only = TRUE))
```

Comment:
This would be a bit clearer: 
`reqs <- vapply(pkgs, require, character.only = TRUE, FUN.VALUE = logical(1))`

(and simplifies the following line)